services/paymaster/migrations/20251020133349_init.up.sql

```diff
@@ -1,4 +1,3 @@
-CREATE TYPE app_role AS ENUM ('owner', 'admin');
```

Comment:
it seems we didn't have some of these in the down migration. we should be careful about making sure `down` composed with `up` leads back to the original state, i.e.

$$ \text{down} \circ \text{up} = \text{id}$$

Reply:
we should generally test this as part of the vetting process for a PR. let's do this before we merge this PR, to make sure that the down and up migrations are truly inverses.